source/assets/js/playground/autocomplete.ts

```diff
@@ -0,0 +1,245 @@
+const moduleUseRegex = new RegExp(/['"]sass:(?<modName>.*)['"]/);
+
+// Completion results for built in Sass modules following the `sass:` namespace.
+const moduleCompletions = Object.freeze(
+  moduleMetadata.map(mod => ({
+    label: `sass:${mod.name}`,
+    apply: `sass:${mod.name}`,
+    info: mod.description,
+    type: 'class',
+    validFor: identifier,
+    _moduleName: mod.name as ModuleName,
+  }))
+);
+
+// Completions for the import of built in modules, for instance "sass:color".
+function moduleMetadataCompletion(
+  context: CompletionContext
+): CompletionResult | null {
+  const nodeBefore = syntaxTree(context.state).resolveInner(context.pos, -1);
+  const potentialTypes = [
+    'StringLiteral', // When wrapped in quotes- `"sass:"`
+    'ValueName', // No end quote, before semicolon- `"sa`
+    ':', // No end quote, on semicolon- `"sass:`
+    'PseudoClassName', // No end quote, after semicolon- `"sass:m`
+  ];
+  if (!potentialTypes.includes(nodeBefore.type.name)) return null;
+  const potentialParentTypes = [
+    'UseStatement', // When wrapped in quotes
+    'PseudoClassSelector', // No end quote, after the colon
+  ];
+  if (!potentialParentTypes.includes(nodeBefore.parent?.type.name || '')) {
+    return null;
+  }
+
+  const moduleMatch = context.matchBefore(/["'](sass:)?\w*/);
+
+  if (!moduleMatch) return null;
+  if (moduleMatch.from === moduleMatch.to && !context.explicit) return null;
+
+  const included = includedmoduleMetadata(context.state);
+  const notIncludedModuleCompletions = moduleCompletions.filter(
+    moduleCompletion => !included.includes(moduleCompletion._moduleName)
+  );
+  return {
+    from: moduleMatch.from + 1,
+    to: moduleMatch.to,
+    options: notIncludedModuleCompletions,
+    validFor: identifier,
+  };
+}
+
+// Completion results for module variables.
+const moduleVariableCompletions = Object.freeze(
+  moduleMetadata.reduce(
+    (acc: {[k: string]: CompletionResult['options'] | []}, mod) => {
+      acc[mod.name] =
+        mod.variables?.map(variable => ({
+          label: `${mod.name}.${variable}`,
+          type: 'variable',
+        })) || [];
+      return acc;
+    },
+    {}
+  )
+);
+
+// Completion results for module functions.
+const moduleFunctionsCompletions = Object.freeze(
+  moduleMetadata.reduce(
+    (acc: {[k: string]: CompletionResult['options'] | []}, mod) => {
+      acc[mod.name] =
+        mod.functions.map(func => ({
+          label: `${mod.name}.${func}`,
+          apply: `${mod.name}.${func}(`,
+          type: 'method',
+          boost: 10,
+          validFor: identifier,
+        })) || [];
+      return acc;
+    },
+    {}
+  )
+);
+
+// Type predicate for modules names.
+function isModuleName(string?: string | null): string is ModuleName {
+  return moduleNames.includes(string as ModuleName);
+}
+
+// Returns the list of built in modules that are included in the text.
+function includedmoduleMetadata(state: EditorState): ModuleName[] {
```

Comment:
```suggestion
function includedModuleMetadata(state: EditorState): ModuleName[] {
```

Reply:
Fixed in https://github.com/sass/sass-site/pull/1166/commits/7c4ef88299d155c3d4db97b36d502220278162a6